test/pulp_api_test.rb

```diff
@@ -36,4 +36,18 @@ def test_returns_50X_on_socket_error
   ensure
     Net::HTTP.any_instance.unstub(:request)
   end
-end
\ No newline at end of file
+
+  def test_returns_pulp_disk_on_200
+    get '/status/diskfree'
+    response = JSON.parse(last_response.body)
+    assert last_response.ok?, "Last response was not ok: #{last_response.body}"
+    assert_equal(%w(path free total percent_used size), response.keys)
+  end
+
+  def test_returns_500_on_path_invalid
```

Comment:
Again, I suggest validating paths before the plugin is loaded, in which case this test won't be necessary.